### PR TITLE
Libpitt/commons

### PR DESCRIPTION
**Pull request by libpitt**
Change log:
1. Use ubkg for organ and entity types

### CODE REVIEW

File: src/sennet_translator.py
Comment (libpitt) [inline src/sennet_translator.py:1]: @shirey @yuanzhou @maxsibilla

Comment (yuanzhou) [reply src/sennet_translator.py:1]: For reference: https://github.com/dbmi-pitt/search-adaptor/pull/33

**Comment by libpitt** [comment]
@maxsibilla 
I am a bit perplexed by what I am seeing here. Adding `SNT952.MQJJ.837` as ancestor during Dataset registration doesn't show a value in the `Subtype` column. I notice a `display_subtype` property in the response from search-api for other entities, but not `SNT952.MQJJ.837`. Is this expected behavior? Also, where is the formatting for `{display_subtype}` being handled? Thanks.

**Comment by maxsibilla** [comment]
@libpitt The `display_subtype` field is generated in the search api here: https://github.com/sennetconsortium/search-api/blob/d80a2db303cff50646dd0143d018ebe2d27b7068/src/sennet_translator.py#L487-L516

The function `get_type_description` exists inside the Search Adaptor submodule (https://github.com/dbmi-pitt/search-adaptor/blob/a556a8ec0ea299b67c8d9ab0f0feba54a4208d35/src/translator/tranlation_helper_functions.py#L60) and currently utilizes yaml files to help generate these `display_subtype` fields. We can bypass calling this function altogether though with the UBKG integration.

What you are seeing with `SNT952.MQJJ.837` not having a subtype is not the expected behavior. I believe we would expect to see the `sample_category` here. We can issue a reindex by calling a PUT against this endpoint once we fix the code: https://search-api.dev.sennetconsortium.org/reindex-all. The brackets seem to erroneously added in `get_type_description` but again we shouldn't need to call that function anymore so hopefully that will resolve it.

**Comment by maxsibilla** [comment]
@libpitt The method `generate_display_subtype` is only called by the search-api during a reindex or reindex-all call. During the reindex process a document is generated for an entity and then pushed to the elasticsearch cluster. Whenever we are interacting with an entity in the portal we are working with that document in JSON format so `display_subtype` is already created

**Review by maxsibilla — APPROVED** [review]

**Pull request merged**